tests/cli/workflow/balsamic/test_cli_balsamic_config_case.py

```diff
@@ -515,3 +515,43 @@ def test_unknown_sex_is_set_as_unknown(
 
     # THEN the sex is set to unknown in the command
     assert "--gender unknown" in caplog.text
+
+    def test_head_job_partition_flag_included_in_cli_command(
```

Comment:
Lets add the new assertion of line 557 to the happy path tests (single_wgs, single_panel, paired_wgs_paired_panel) instead of making a new test for this assertion